benchmarks/torchbench_model.py

```diff
@@ -209,6 +211,25 @@ def set_up(self):
     del benchmark
     gc.collect()
 
+  def apply_default_precision_config(self, test, benchmark):
```

Comment:
Thanks for finding this and adding it to the benchmarks. 
I am wondering if this is the right place to fix this. 
Is this something we can add in the dynamo bridge when lowering to XLA?

Reply:
This is a workaround. See b/316126405

Reply:
I wonder if we should add these things to benchmarking or if that is just more confusing overall. We should at least add a comment in these cases.